Please review it first before merging!!! easy_install was commented out or replaced with pip.

Muntasib-creator opened the pull request.

Muntasib-creator committed:
easy_install was commented out or replaced with pip

riz-hossain reviewed (changes requested):
Remove any debug print statements

riz-hossain commented on ZeuZ_Node_Installer.py:
```diff
@@ -163,7 +163,7 @@ def get_required_mods():
                 quit()
-
+            print("Asche")
             install(type="sudo", cmd="easy_install pip")
```
Remove debuging print

Muntasib-creator replied:
Done removing

Muntasib-creator replied:
Dont merge it. Its not been tested. Just solved all the issues on my laptop so just saved the codes in separate branch

Muntasib-creator committed:
removed debugging line

Muntasib-creator committed:
Revert "easy_install was commented out or replaced with pip"

This reverts commit f64e0ed1

Muntasib-creator committed:
[Fix] Windows os fixes:
1. Removed easy_insall
2. connect pip install with requirements.txt of latest beta
3. download latest beta Node
4. download nodejs 14
5. Removed npm folder before installing nodejs
6. psutil, pywin32, ida, pythonnet fixed
7. Removed node_gui shortcut
8. Fixed node_cli shortcut bug

Muntasib-creator committed:
linux core fixes
